server/plugin/client.go

```diff
@@ -24,6 +24,7 @@ type Client interface {
 	CreateSubscription(body *serializers.CreateSubscriptionRequestPayload, project *serializers.ProjectDetails, channelID, pluginURL, mattermostUserID string) (*serializers.SubscriptionValue, int, error)
 	DeleteSubscription(organization, subscriptionID, mattermostUserID string) (int, error)
 	CheckIfUserIsProjectAdmin(organizationName, projectID, pluginURL, mattermostUserID string) (int, error)
+	GetGitRepositories(organization, projectName, mattermostUserID string) (*serializers.GitRepositoriesResponse, int, error)
```

Comment:
```suggestion
	GetRepositories(organization, projectName, mattermostUserID string) (*serializers.GitRepositoriesResponse, int, error)
```
I think we should not use `git` anywhere as it gives the impression of using `Github`. Are there multiple types of repositories?

Reply:
This is done intentionally because Azure DevOps Repos has support for different types of repositories but we need to target only GIT for now so, to make it descriptive for devs we have added Git as a prefix/suffix, in future if we need to add support for others then this could be helpful to differentiate.

Reply:
Also, Azure DevOps API used here has mentioned this in the URL `_apis/git/repositories?api-version=6.0`